src/lang/lang_pp.ml

```diff
@@ -261,12 +261,13 @@ let parse_comments tokenizer =
            (fun (n, _) -> String.length n = 0 || n.[0] <> '_')
            doc#get_subsections)
     in
-    let rec parse_doc (main, special, params) = function
-      | [] -> (main, special, params)
+    let rec parse_doc (main, special, params, methods) = function
```

Comment:
Might be time to structure this with a record..

Reply:
Yes but well this is only used very locally...